openshift-knative-operator/pkg/monitoring/rbac_proxy_injection.go

```diff
@@ -99,6 +99,7 @@ func InjectRbacProxyContainer(deployments sets.String, cfg base.ConfigMapData) m
 					"--tls-cert-file=" + filepath.Join(mountPath, "tls.crt"),
 					"--tls-private-key-file=" + filepath.Join(mountPath, "tls.key"),
 					"--logtostderr=true",
+					"--http2-disable",
```

Comment:
wouldn't that... default to this behavior anyways?

Reply:
By this change I don't so

https://github.com/openshift/kube-rbac-proxy/commit/ae32bfa0f11ae2854339ab9af83c44ee1c04fe31#diff-193f95684186b3f36e5beb9ee139212fd891ddbce3f57b79caa5cf328ad4c23aR121

Reply:
I think you are right, we need that flag

Reply:
Exactly the opposite how we do it in Knative. which is ok...